chess/backend/src/game/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import logic
+
+urlpatterns = [
+    path("", logic.create_chess_game, name="gaston-is-wrong"),
```

Comment:
gaston -> jacob

Reply:
yo mamma services more pull requests than github

Reply:
For all that squashing and mergin yo mamma does, I am surprised you are a rebase guy